src/op_mode/ntp.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2024 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import sys
+import typing
+
+import vyos.opmode
+from vyos.configquery import ConfigTreeQuery
+from vyos.utils.process import cmd
+
+def _get_formatted_output(output: str) -> dict:
+
+    # Initialize dictionary keys to match output of
+    # chrony commmand being passed into this function.
+    # From some commands, the -c switch outputs more
+    # parameters, make sure to include them all below.
+    # See to chronyc(1) for definition of key variables
+    match output:
+        case "chronyc -c 'activity'":
+            keys: list = [
+            'Sources_online',
+            'Sources_offline',
+            'Sources_doing_burst_(return-to-online)',
+            'Sources_doing_burst_(return-to-offline)',
+            'Sources_with_unknown_address'
+            ]
+
+        case "chronyc -c 'sources'":
+            keys: list = [
+            'M',
+            'S',
+            'Name/IP Address',
```

Comment:
Imagine that you parse the raw data dictionary 
The key “Name/ IP Address” really looks strange, even after normalization it looks ugly.
I’m expecting lowercase for keys without spec symbols.
Other looks good.
Thanks!

Reply:
Understood, will fix :)